pkg/client/informers/externalversions/build/v1alpha1/builder.go

```diff
@@ -22,10 +22,10 @@ import (
 	"context"
 	time "time"
 
-	buildv1alpha1 "github.com/pivotal/kpack/pkg/apis/build/v1alpha1"
+	buildapi "github.com/pivotal/kpack/pkg/apis/build/v1alpha1"
```

Comment:
This file is autogenerated so we should leave it unmodified.

Reply:
> This file is autogenerated so we should leave it unmodified.

I've reset `pkg/client` to `main`.